tests/test_cube/test_cube.py

```diff
@@ -142,15 +141,9 @@ def test_segy_export_import(tmp_path, testdata_path):
 def test_storm_import(testdata_path):
     """Import Cube using Storm format (case Reek)."""
 
-    st1 = xtg.timer()
```

Comment:
When measuring time is needed (often while developing), the @functimer approach is much better.